tests/testthat/test-tokens_select.R

```diff
@@ -218,43 +218,43 @@ test_that("tokens_select works as expected for bigrams selected on char, list of
     )
 })
 
-test_that("tokens_select works correctly with collocations objects", {
-    expect_equal(
-        as.list(tokens_select(toks_uni, coll_bi$collocation)),
-        list(d1 = character(), d2 = character())
-    )
-    expect_equal(
-        as.list(tokens_remove(toks_uni, phrase(coll_bi$collocation))),
-        list(d1 = c("c", "d"), d2 = c("i", "j"))
-    )
-    expect_equal(
-        as.list(tokens_remove(toks_uni, coll_bi)),
-        as.list(tokens_remove(toks_uni, phrase(coll_bi$collocation)))
-    )
-    expect_equal(
-        as.list(tokens_select(toks_uni, coll_tri$collocation)),
-        list(d1 = character(), d2 = character())
-    )
-    expect_equal(
-        as.list(tokens_select(toks_bi, coll_bi$collocation)),
-        list(d1 = c("a b", "e g", "g h"), d2 = c("a b", "e g", "g h"))
-    )
-    expect_equal(
-        as.list(tokens_select(toks_bi, phrase(coll_bi$collocation))),
-        list(d1 = character(), d2 = character())
-    )
-    expect_equal(
-        as.list(tokens_select(toks_bi, phrase(coll_bi$collocation))),
-        as.list(tokens_select(toks_bi, coll_bi))
-    )
-    expect_equal(
-        as.list(tokens_select(toks_bi, coll_tri)),
-        list(d1 = character(), d2 = character())
-    )
-    expect_silent(
-        tokens_select(toks_bi, coll_tri)
-    )
-})
+# test_that("tokens_select works correctly with collocations objects", {
```

Comment:
We should test using saved collocation objects.

Reply:
Added back in 9e9aa94.